recipes/libassert/v2/conanfile.py

```diff
@@ -53,7 +53,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("cpptrace/0.5.3", transitive_headers=True, transitive_libs=True)
```

Comment:
Bumping cpptrace dependency while I'm here

Reply:
Approving because noone other library is using cpptrace so there can be no conflicts, but we need to take care in the future if it proliferates to at least keep them in sync (Or until a time we feel confident enought to use version ranges everywhere!)